src/latbin/lattice-compose.cc

```diff
@@ -39,22 +39,34 @@ int main(int argc, char *argv[]) {
         "or lattices with FSTs (rspecifiers are assumed to be lattices, and\n"
         "rxfilenames are assumed to be FSTs, which have their weights interpreted\n"
         "as \"graph weights\" when converted into the Lattice format.\n"
+        "Or, rspecifier can be ark of biasing FSTs, see --compose-with-fst=true.\n"
```

Comment:
This whole message is not very clear, especially with the modification.  Can write:

 Depending on the command-line arguments, either composes lattices with lattices,
 or lattices with a single FST or multiple FSTs (whose weights are interpreted as "graph weights").

Then change the lines below to:
```
Usage: lattice-compose [options] <lattice-rspecifier1> "
        "<lattice-rspecifier2|fst-rxfilename2|fst-rspecifier2> <lattice-wspecifier>\n"
 If the 2nd arg is an rspecifier, it is interpreted by default as a table of lattices,
 or as a table of FSTs if you specify --compose-with-fst=true.
```